[2014-03-19 10:08:46] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:159 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(159): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 159, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(63): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-19 10:10:03] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:159 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(159): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 159, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(63): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-19 10:11:32] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:159 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(159): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 159, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(63): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-19 10:13:28] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Controllers\Admin\Merchant' not found' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:143 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 10:14:26] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::paramval()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('paramval', Array) #1 [internal function]: Illuminate\Database\Query\Builder->paramval('percentage') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('paramval', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->paramval('percentage') #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2653): call_user_func_array(Array, Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(216): Illuminate\Database\Eloquent\Model->__call('paramval', Array) #7 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(216): Merchant->paramval('percentage') #8 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(64): Controllers\Admin\MainvoiceController->generate() #9 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #20 {main} [] []
[2014-03-19 10:15:24] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::paramval()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('paramval', Array) #1 [internal function]: Illuminate\Database\Query\Builder->paramval('percentage') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('paramval', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->paramval('percentage') #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/Relation.php(271): call_user_func_array(Array, Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(216): Illuminate\Database\Eloquent\Relations\Relation->__call('paramval', Array) #7 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(216): Illuminate\Database\Eloquent\Relations\BelongsTo->paramval('percentage') #8 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(64): Controllers\Admin\MainvoiceController->generate() #9 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #20 {main} [] []
[2014-03-19 10:15:56] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::paramval()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('paramval', Array) #1 [internal function]: Illuminate\Database\Query\Builder->paramval('percentage') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('paramval', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->paramval('percentage') #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/Relation.php(271): call_user_func_array(Array, Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(216): Illuminate\Database\Eloquent\Relations\Relation->__call('paramval', Array) #7 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(216): Illuminate\Database\Eloquent\Relations\BelongsTo->paramval('percentage') #8 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(64): Controllers\Admin\MainvoiceController->generate() #9 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #20 {main} [] []
[2014-03-19 10:16:19] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::paramval()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('paramval', Array) #1 [internal function]: Illuminate\Database\Query\Builder->paramval('percentage') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('paramval', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->paramval('percentage') #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/Relation.php(271): call_user_func_array(Array, Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(152): Illuminate\Database\Eloquent\Relations\Relation->__call('paramval', Array) #7 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(152): Illuminate\Database\Eloquent\Relations\BelongsTo->paramval('percentage') #8 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(64): Controllers\Admin\MainvoiceController->generate() #9 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #20 {main} [] []
[2014-03-19 10:16:22] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::paramval()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('paramval', Array) #1 [internal function]: Illuminate\Database\Query\Builder->paramval('percentage') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('paramval', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->paramval('percentage') #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/Relation.php(271): call_user_func_array(Array, Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(152): Illuminate\Database\Eloquent\Relations\Relation->__call('paramval', Array) #7 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(152): Illuminate\Database\Eloquent\Relations\BelongsTo->paramval('percentage') #8 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(64): Controllers\Admin\MainvoiceController->generate() #9 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #20 {main} [] []
[2014-03-19 10:19:13] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:164 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(164): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 164, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(64): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-19 10:21:19] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:164 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(164): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 164, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(64): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-19 10:23:14] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0:120 Stack trace: #0 /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0(120): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 120, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-19 10:27:34] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::merchantagreement()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('merchantagreeme...', Array) #1 [internal function]: Illuminate\Database\Query\Builder->merchantagreement(1033) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('merchantagreeme...', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->merchantagreement(1033) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2653): call_user_func_array(Array, Array) #6 /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0(120): Illuminate\Database\Eloquent\Model->__call('merchantagreeme...', Array) #7 /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0(120): Mainvoice->merchantagreement(1033) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #13 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #18 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #20 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #21 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #22 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #23 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #24 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #25 {main} [] []
[2014-03-19 10:27:58] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0:120 Stack trace: #0 /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0(120): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 120, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-19 10:28:48] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0:120 Stack trace: #0 /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0(120): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 120, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-19 10:30:03] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0:120 Stack trace: #0 /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0(120): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 120, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-19 10:32:07] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:32:57] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:34:10] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:36:38] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:36:49] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:39:25] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Database\Eloquent\Collection::getResults()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:2070 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 10:39:34] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function get() on a non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 10:40:40] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::merchantagreement()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('merchantagreeme...', Array) #1 [internal function]: Illuminate\Database\Query\Builder->merchantagreement() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('merchantagreeme...', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->merchantagreement() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2653): call_user_func_array(Array, Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Database\Eloquent\Model->__call('merchantagreeme...', Array) #7 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Mainvoice->merchantagreement() #8 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #12 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #19 {main} [] []
[2014-03-19 10:43:16] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:43:31] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:43:40] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:44:06] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:44:58] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:45:16] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:46:41] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:47:04] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Pagination\Paginator::$name' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:68 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(68): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 68, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:47:33] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Pagination\Paginator::merchantagreement()' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:68 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 10:47:52] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Pagination\Paginator::$merchantagreement' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:68 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(68): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 68, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 10:58:59] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0:120 Stack trace: #0 /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0(120): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 120, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-19 10:59:07] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0:120 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 10:59:49] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:282 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(282): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 282, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1033') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-19 11:00:26] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0:120 Stack trace: #0 /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0(120): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 120, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-19 11:02:13] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:02:30] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:03:03] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 69, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:03:11] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:70 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 70, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:03:19] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:70 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 70, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:03:23] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::getResults()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('getResults', Array) #1 [internal function]: Illuminate\Database\Query\Builder->getResults() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('getResults', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->getResults() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2653): call_user_func_array(Array, Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2070): Illuminate\Database\Eloquent\Model->__call('getResults', Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2070): Merchant->getResults() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2552): Illuminate\Database\Eloquent\Model->getAttribute('merchant') #9 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Database\Eloquent\Model->__get('merchant') #10 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #14 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #20 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #21 {main} [] []
[2014-03-19 11:04:22] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function get() on a non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 11:07:57] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Pagination\Paginator::merchant()' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:68 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 11:11:25] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'exit' (T_EXIT)' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:69 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 11:11:31] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::getResults()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('getResults', Array) #1 [internal function]: Illuminate\Database\Query\Builder->getResults() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('getResults', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->getResults() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2653): call_user_func_array(Array, Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2070): Illuminate\Database\Eloquent\Model->__call('getResults', Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2070): Merchant->getResults() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2552): Illuminate\Database\Eloquent\Model->getAttribute('merchant') #9 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(68): Illuminate\Database\Eloquent\Model->__get('merchant') #10 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #14 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #20 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #21 {main} [] []
[2014-03-19 11:11:42] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:68 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(68): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 68, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:11:51] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::getResults()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('getResults', Array) #1 [internal function]: Illuminate\Database\Query\Builder->getResults() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('getResults', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->getResults() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2653): call_user_func_array(Array, Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2070): Illuminate\Database\Eloquent\Model->__call('getResults', Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2070): Merchant->getResults() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2552): Illuminate\Database\Eloquent\Model->getAttribute('merchant') #9 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(68): Illuminate\Database\Eloquent\Model->__get('merchant') #10 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #14 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #20 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #21 {main} [] []
[2014-03-19 11:14:13] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::getResults()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('getResults', Array) #1 [internal function]: Illuminate\Database\Query\Builder->getResults() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('getResults', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->getResults() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2653): call_user_func_array(Array, Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2070): Illuminate\Database\Eloquent\Model->__call('getResults', Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2070): Merchant->getResults() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2552): Illuminate\Database\Eloquent\Model->getAttribute('merchant') #9 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Database\Eloquent\Model->__get('merchant') #10 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #14 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #20 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #21 {main} [] []
[2014-03-19 11:14:22] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:71 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(71): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 71, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:14:54] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function Controllers\Admin\car_dump()' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:71 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 11:15:00] log.ERROR: exception 'ErrorException' with message 'Invalid argument supplied for foreach()' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:70 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Exception\Handler->handleError(2, 'Invalid argumen...', '/var/www/arp_de...', 70, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:16:29] log.ERROR: exception 'ErrorException' with message 'Invalid argument supplied for foreach()' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:71 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(71): Illuminate\Exception\Handler->handleError(2, 'Invalid argumen...', '/var/www/arp_de...', 71, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:21:29] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:70 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 70, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:21:51] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:70 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 70, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:21:56] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:70 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 70, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:22:15] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:70 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 70, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:23:29] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:70 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 70, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:25:27] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0:120 Stack trace: #0 /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0(120): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 120, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-19 11:26:37] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0:120 Stack trace: #0 /var/www/arp_dev/app/storage/views/bb246097065f941c3fdcdc78accd32c0(120): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 120, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-19 11:27:03] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:70 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 70, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:27:15] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:70 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 70, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:27:23] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:70 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(70): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 70, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-19 11:37:08] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:282 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(282): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 282, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1033') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-19 12:53:48] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/34701f2e9519f64aa8936ae800168eb2:302 Stack trace: #0 /var/www/arp_dev/app/storage/views/34701f2e9519f64aa8936ae800168eb2(302): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 302, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getEdit', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getEdit', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1038') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-19 15:04:50] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42703]: Undefined column: 7 ERROR:  column "merchant_id" does not exist LINE 1: select min("id") as aggregate from "order" where "merchant_i...                                                          ^' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:274 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(274): PDOStatement->execute(Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(521): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\PostgresConnection), 'select min("id"...', Array) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select min("id"...', Array, Object(Closure)) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select min("id"...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1369): Illuminate\Database\Query\Builder->get(Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1322): Illuminate\Database\Query\Builder->aggregate('min', Array) #8 [internal function]: Illuminate\Database\Query\Builder->min('id') #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #10 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(658): Illuminate\Database\Eloquent\Builder->__call('min', Array) #11 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(658): Illuminate\Database\Eloquent\Builder->min('id') #12 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(191): Controllers\Admin\MainvoiceController->getTransactionRange(11, '2014-03-10 00:0...', '2014-03-17 23:5...') #13 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #14 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #18 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #20 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #21 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #22 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #23 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #24 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #25 {main} Next exception 'Exception' with message 'SQLSTATE[42703]: Undefined column: 7 ERROR:  column "merchant_id" does not exist LINE 1: select min("id") as aggregate from "order" where "merchant_i...                                                          ^ (SQL: select min("id") as aggregate from "order" where "merchant_id" = ? and "datetimecreated" >= ? and "datetimecreated" <= ?) (Bindings: array (   0 => 11,   1 => '2014-03-10 00:00:00',   2 => '2014-03-17 23:59:59', ))' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:556 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(529): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select min("id"...', Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select min("id"...', Array, Object(Closure)) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select min("id"...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1369): Illuminate\Database\Query\Builder->get(Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1322): Illuminate\Database\Query\Builder->aggregate('min', Array) #7 [internal function]: Illuminate\Database\Query\Builder->min('id') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #9 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(658): Illuminate\Database\Eloquent\Builder->__call('min', Array) #10 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(658): Illuminate\Database\Eloquent\Builder->min('id') #11 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(191): Controllers\Admin\MainvoiceController->getTransactionRange(11, '2014-03-10 00:0...', '2014-03-17 23:5...') #12 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #13 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #17 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #20 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #21 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #22 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #23 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #24 {main} [] []
[2014-03-19 15:11:33] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42702]: Ambiguous column: 7 ERROR:  column reference "id" is ambiguous LINE 1: select min("id") as aggregate from "order" inner join "trans...                    ^' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:274 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(274): PDOStatement->execute(Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(521): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\PostgresConnection), 'select min("id"...', Array) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select min("id"...', Array, Object(Closure)) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select min("id"...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1369): Illuminate\Database\Query\Builder->get(Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1322): Illuminate\Database\Query\Builder->aggregate('min', Array) #8 [internal function]: Illuminate\Database\Query\Builder->min('id') #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #10 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(659): Illuminate\Database\Eloquent\Builder->__call('min', Array) #11 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(659): Illuminate\Database\Eloquent\Builder->min('id') #12 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(191): Controllers\Admin\MainvoiceController->getTransactionRange(11, '2014-03-10 00:0...', '2014-03-17 23:5...') #13 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #14 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #18 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #20 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #21 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #22 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #23 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #24 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #25 {main} Next exception 'Exception' with message 'SQLSTATE[42702]: Ambiguous column: 7 ERROR:  column reference "id" is ambiguous LINE 1: select min("id") as aggregate from "order" inner join "trans...                    ^ (SQL: select min("id") as aggregate from "order" inner join "transaction" on "transaction"."id" = "order"."transaction_id" where "transaction"."merchant_id" = ? and "order"."datetimecreated" >= ? and "order"."datetimecreated" <= ?) (Bindings: array (   0 => 11,   1 => '2014-03-10 00:00:00',   2 => '2014-03-17 23:59:59', ))' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:556 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(529): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select min("id"...', Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select min("id"...', Array, Object(Closure)) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select min("id"...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1369): Illuminate\Database\Query\Builder->get(Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1322): Illuminate\Database\Query\Builder->aggregate('min', Array) #7 [internal function]: Illuminate\Database\Query\Builder->min('id') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #9 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(659): Illuminate\Database\Eloquent\Builder->__call('min', Array) #10 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(659): Illuminate\Database\Eloquent\Builder->min('id') #11 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(191): Controllers\Admin\MainvoiceController->getTransactionRange(11, '2014-03-10 00:0...', '2014-03-17 23:5...') #12 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #13 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #17 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #20 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #21 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #22 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #23 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #24 {main} [] []
[2014-03-19 15:12:20] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42702]: Ambiguous column: 7 ERROR:  column reference "amount" is ambiguous LINE 1: select sum("amount") as aggregate from "order" inner join "t...                    ^' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:274 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(274): PDOStatement->execute(Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(521): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\PostgresConnection), 'select sum("amo...', Array) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select sum("amo...', Array, Object(Closure)) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select sum("amo...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1369): Illuminate\Database\Query\Builder->get(Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1344): Illuminate\Database\Query\Builder->aggregate('sum', Array) #8 [internal function]: Illuminate\Database\Query\Builder->sum('amount') #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #10 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(677): Illuminate\Database\Eloquent\Builder->__call('sum', Array) #11 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(677): Illuminate\Database\Eloquent\Builder->sum('amount') #12 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(191): Controllers\Admin\MainvoiceController->getTransactionRange(11, '2014-03-10 00:0...', '2014-03-17 23:5...') #13 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #14 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #18 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #20 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #21 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #22 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #23 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #24 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #25 {main} Next exception 'Exception' with message 'SQLSTATE[42702]: Ambiguous column: 7 ERROR:  column reference "amount" is ambiguous LINE 1: select sum("amount") as aggregate from "order" inner join "t...                    ^ (SQL: select sum("amount") as aggregate from "order" inner join "transaction" on "transaction"."id" = "order"."transaction_id" where "transaction"."merchant_id" = ? and "order"."datetimecreated" >= ? and "order"."datetimecreated" <= ?) (Bindings: array (   0 => 11,   1 => '2014-03-10 00:00:00',   2 => '2014-03-17 23:59:59', ))' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:556 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(529): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select sum("amo...', Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select sum("amo...', Array, Object(Closure)) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select sum("amo...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1369): Illuminate\Database\Query\Builder->get(Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1344): Illuminate\Database\Query\Builder->aggregate('sum', Array) #7 [internal function]: Illuminate\Database\Query\Builder->sum('amount') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #9 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(677): Illuminate\Database\Eloquent\Builder->__call('sum', Array) #10 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(677): Illuminate\Database\Eloquent\Builder->sum('amount') #11 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(191): Controllers\Admin\MainvoiceController->getTransactionRange(11, '2014-03-10 00:0...', '2014-03-17 23:5...') #12 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #13 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #17 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #20 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #21 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #22 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #23 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #24 {main} [] []
[2014-03-19 15:41:44] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$conv_rate' (T_VARIABLE)' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:219 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 15:42:07] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$costrows' (T_VARIABLE)' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:223 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 15:42:35] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Unsupported operand types' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:217 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 15:48:44] log.ERROR: exception 'ErrorException' with message 'Undefined variable: numcb' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:215 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(215): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 215, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-19 15:50:06] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Unsupported operand types' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:219 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-19 15:50:45] log.ERROR: exception 'ErrorException' with message 'Array to string conversion' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:219 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(219): Illuminate\Exception\Handler->handleError(8, 'Array to string...', '/var/www/arp_de...', 219, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-19 16:11:46] log.ERROR: exception 'ErrorException' with message 'Undefined variable: deducted_amounts' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:226 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(226): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 226, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
